File: NERP.Admin.Web1/ClientApp/src/app/accessories/models/accessory.ts
import { DbBaseEntity } from '../../common/models/dbBaseEntity';
import { CustomField } from '../../common/models/customField';
import { FileAttachment } from '../../common/models/FileAttachment';
import { Supplier } from '../../common/models/supplier';

export class Accessory extends DbBaseEntity {
    name: string;
    code?: string;
    icon?: string;
    parentId?: string;
    parentEntityType?: string;
    type?: string;
    brand?: string;
    model?: string;
    zoomLevel: number;
    customFields?: CustomField[] = new Array<CustomField>();
    photos?: FileAttachment;
    attachments?: FileAttachment;
    suppliers?: Supplier[] = new Array<Supplier>();
}

File: NERP.NetworkEntities/Models/Accessory.cs
using Common.DataAccess.Mongo;
using NERP.Common.Models;
using System.Collections.Generic;

namespace NERP.NetworkEntities.Models
{
    public class Accessory : DbBaseEntity
    {
        public string Name { get; set; }
        public string Code { get; set; }
        public string Icon { get; set; }
        public string ParentId { get; set; }
        public string ParentEntityType { get; set; }
        public string Type { get; set; }
        public string Brand { get; set; }
        public string Model { get; set; }
        public int ZoomLevel { get; set; }
        public List<FileAttachment> Photos { get; set; }
        public List<FileAttachment> Attachments { get; set; }
        public List<Supplier> Suppliers { get; set; }

    }
}
